roles/base/tasks/packages.yml

```diff
@@ -26,7 +27,7 @@
     - packages
 
 - name: Install cower
-  aur: name=cower user={{ user.name }} skip_pgp=yes
+  pacman: name=cower state=present
```

Comment:
ARM folks are doing a little trick here, using [this repo](https://github.com/archlinuxarm/PKGBUILDs) as the basis to build certain ARM-customized packages and [mirror them](http://mirror.archlinuxarm.org/).  A couple of things like cower are there. They are pulled into regular pacman via the `[aur]` and `[alarm]` sections of pacman.conf.